faithcomesbyhearing/dbp
# FCBH-1312 Clean up migrations

# Description
- Refactored all the migrations
- Moved the migrations that change tables to the creation migrations that create tables

## Issue Link
Original Story: [FCBH-1312](https://fullstacklabs.atlassian.net/browse/FCBH-1312) 
Review Story: [FCBH-1366](https://fullstacklabs.atlassian.net/browse/FCBH-1366) 

## How Do I QA This
- On your local environment create 2 new databases for example:
    - `dbp_users_2`
    - `dbp_prod_2`
- Modify the `.env` databases values
- On your local environment run `php artisan migrate --database="dbp_users"` to run the migrations
- Import SQL dump data of the most recent databases
- Verify that the migration worked


## Checklist
